config/datatypes_conf.xml.sample

```diff
@@ -443,8 +443,10 @@
     <sniffer type="galaxy.datatypes.tabular:Vcf"/>
     <sniffer type="galaxy.datatypes.binary:TwoBit"/>
     <sniffer type="galaxy.datatypes.binary:GeminiSQLite"/>
-    <sniffer type="galaxy.datatypes.proteomics:MzSQlite"/>
+    <sniffer type="galaxy.datatypes.binary:MzSQlite"/>
```

Comment:
Wouldn't that also be true for my IdpDB change from proteomics to binary? If so, better to kill 2 birds with 1 merge, I suppose. My understanding is I that I have to move them to make the sniffer order work.